lib/stackprof/report.rb

```diff
@@ -1,5 +1,6 @@
 require 'pp'
 require 'digest/md5'
+require 'json'
```

Comment:
Could we move this into the `print_json` method so we are only requiring `JSON` when we are using it?

Reply:
Yep; makes sense.

Done